bin/prepare-release.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
```

Comment:
Should this be set executable?

Reply:
I'm not sure. This is the exact same script/mode we've been using over in the [glean repo](https://github.com/mozilla/glean/blob/main/bin/prepare-release.sh).